rust/operator-binary/src/controller.rs

```diff
@@ -965,10 +972,23 @@ async fn build_node_rolegroup_statefulset(
         }
     })?;
 
+    let nifi_args = vec![formatdoc! {"
+            {COMMON_BASH_TRAP_FUNCTIONS}
+            {remove_vector_shutdown_file_command}
+            prepare_signal_handlers
+            bin/nifi.sh run &
+            wait_for_termination $!
+            {create_vector_shutdown_file_command}
+            ",
+    remove_vector_shutdown_file_command =
+        remove_vector_shutdown_file_command(STACKABLE_LOG_DIR),
+    create_vector_shutdown_file_command =
+        create_vector_shutdown_file_command(STACKABLE_LOG_DIR),
+    }];
     let container_nifi = container_builder
         .image_from_product_image(resolved_product_image)
         .command(vec!["/bin/bash".to_string(), "-c".to_string()])
```

Comment:
This could be extended to:
```
"/bin/bash".to_string(),
                            "-x".to_string(),
                            "-euo".to_string(),
                            "pipefail".to_string(),
                            "-c".to_string(),
```

Reply:
https://github.com/stackabletech/nifi-operator/pull/528/commits/565c5ec38a0b2a4a91550107769aa6c439b6c210

Reply:
Thanks!